backend/api/Models/Concepts/ActivityInstance/PimsActInstPropFile.cs

```diff
@@ -0,0 +1,11 @@
+namespace Pims.Api.Models.Concepts
+{
+    public class PimsActInstPropFile : BaseAppModel
```

Comment:
PimsActInstPropFile reads like an Entity object. Why not the more readable version? Same for the map

Reply:
Sure, I can remove the Pims bit, add the Model